jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -1915,14 +1915,14 @@ public RequestDispatcher getRequestDispatcher(String uriInContext)
                 String contextPath = getContextPath();
                 // uriInContext is canonicalized by HttpURI.
                 HttpURI.Mutable uri = HttpURI.build(uriInContext);
-                String pathInfo = uri.getCanonicalPath();
+                String pathInfo = uri.getDecodedPath();
```

Comment:
I know there's a comment a few lines above about being canonicalized, but can we make it very clear that the `pathInfo` will be _both_ canonicalized _and_ decoded?

Reply:
pathInfo is always decoded.   Plus the code is `pathInfo = uri.getDecodedPath()`?    Not sure what else I can do?